Latest changes - if you can merge then I'll see if i can pull down from your git repo

peglegasus opened the pull request:
Scenario.message mapped, still need to map scenario.dictionary

leetoufong committed:
Merge pull request #1 from peglegasus/main

Merge Clay's changes

leetoufong committed:
Merge pull request #2 from peglegasus/main

regex revision and other fixes

leetoufong committed:
Setting state data

leetoufong committed:
Merge branch 'main' of https://github.com/peglegasus/test-healthbot into peglegasus-main

leetoufong committed:
Merge pull request #4 from leetoufong/peglegasus-main

Peglegasus main

leetoufong committed:
Merge pull request #5 from peglegasus/main

updating how we map results from adaptive cards to scenario

leetoufong committed:
Update readme

leetoufong committed:
Merge in Clay's changes. Yar!

leetoufong commented:
@peglegasus (Clay), I had issues merging in these changes. I went the nuclear route as well. I `git clone` your repository, blew away my repo, and replaced it with yours. You have been added as a contributor. Feel free to no longer use this repo.

I repeat, I did not merge in any of these changes. I just replaced my repo with yours.

leetoufong committed:
Update README.md

leetoufong committed:
Update README.md

leetoufong committed:
Update readme, again...

Commit:
Fixing mapping to get us further with the adaptive cards

leetoufong committed:
Rework async JSON requests. Rework scope to allow for dynamic changing and setting. Add replaceScenario logic.

Commit:
fix for passing scenario args to the next scenario

leetoufong committed:
Update readme again

Commit:
fixing localization mapping to be a little better.

Assigning 'islocalized' property to anything that's been localized and then check to see if it's been done. Also adding some error handling as there was an empty array attached to a msg when I am expecting a list of localized strings.

Commit:
Merge branch 'main' of https://github.com/leetoufong/test-healthbot into main

leetoufong committed:
ASCII art because why not?

leetoufong committed:
Again

leetoufong committed:
and Again

Commit:
Adding in dynamic scenario loader to handle scenario switching.

... removing specific loaders , reordering variable initialization to top.

leetoufong committed:
Remove redundant initHealthBot call

Commit:
Adding in start of multi-choice prompt

Commit:
Merge branch 'main' of https://github.com/leetoufong/test-healthbot into main

leetoufong committed:
Remove static files. These are all generated in the code.

leetoufong committed:
Merge branch 'main' of https://github.com/leetoufong/test-healthbot into main

leetoufong committed:
Slight adjustments

Commit:
handling multi-choice prompt and user response

leetoufong committed:
Add FPO session.logOutcomeEvent()

leetoufong committed:
Add endcap identifier

leetoufong committed:
Slight clean up

leetoufong committed:
Apply devMode logic

leetoufong committed:
More devMode situations. Starting the new chatBot styles similar to VTT.

leetoufong committed:
Fix statement issue where it wasnt appending. More VTT inspired styles

leetoufong committed:
Add CDC logo

leetoufong committed:
Edit hostConfig to up the font size on adpativCrds

leetoufong committed:
Adjust adaptiveCard font sizes

leetoufong committed:
Work in progress on trying to get Action.ShowCard to register an event.

leetoufong committed:
Better scroll to bottom logic. Add more useful comments. Update README to include card type information

leetoufong committed:
Adding more healthbot styles.

leetoufong committed:
Add markdown render to prompts

leetoufong committed:
Add smarter card spacing

leetoufong committed:
More style updates

leetoufong committed:
Bringing in more styles

leetoufong committed:
More documentation

leetoufong committed:
Further update documentation. Add CSS4 variables.

leetoufong committed:
Remove index number from button text

leetoufong committed:
Better scroll to bottom

Commit:
updating doLocalizationMapping to be able to localize values stored in an adaptive card body

leetoufong committed:
Add loader image. Todo: Make loader all css later.

Commit:
added logic to fix scroll position if interaction with adaptive card changes card size

leetoufong committed:
Add more space between chat boxes

leetoufong committed:
Merge branch 'main' of https://github.com/leetoufong/test-healthbot into main

Commit:
scroll to bottom and newmessage behavior started

Commit:
Merge branch 'main' of https://github.com/leetoufong/test-healthbot into main

Commit:
Adding in code for replay test harness

Commit:
second round of test harness changes.

leetoufong committed:
Add disabled (but clicked) button states

leetoufong committed:
Remove disabled btn cursor. Remove 'You said: ' from user response message.

leetoufong committed:
Variable-ize the disabled state color

leetoufong committed:
More style matching

leetoufong committed:
Fix broken container

Commit:
adding in ux scenario, beefing up the test harness

Commit:
Merge branch 'main' of https://github.com/leetoufong/test-healthbot into main

Commit:
Updates to the test harness - adding files for the azure healthbot reader

leetoufong committed:
Update readme

leetoufong committed:
Update readme again

leetoufong committed:
Opps, fix typo
